docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">S = T[idx_start: (idx_stop+m-1)]
QS = core.sliding_dot_product(Q, S)
D = core._mass(Q, S, QS, T_sliding_mu[i], T_sliding_std[i],&nbsp;
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; T_sliding_mu[idx_start:idx_stop], T_sliding_std[idx_start:idx_stop])
</pre><p>Replace with</p><p> </p><pre class="ql-syntax" spellcheck="false">QT = core.sliding_dot_product(Q, T[start : stop+m-1])
D = core._mass(Q, T[start:stop+m-1], QT, M_T[i], Σ_T[i],&nbsp;M_T[start:stop], Σ_T[start:stop])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Is `QT` okay? because I noticed before that you used `QT` (in `MASS`-related functions) when you use the `T` itself and not its slice. Just wanted to do a double check with you here.

Reply:
<p>Yeah, it's reasonable here so it is no problem. I can quickly see that it is:</p><pre class="ql-syntax" spellcheck="false">QT = core.sliding_dot_product(Q, T[start : stop - m + 1])
</pre><p>and not</p><pre class="ql-syntax" spellcheck="false">QT = core.sliding_dot_product(Q, T)
</pre><p> </p>